fury/shaders/billboard_spheres_impl.frag

```diff
@@ -0,0 +1,21 @@
+// camera right
+vec3 uu = vec3(MCVCMatrix[0][0], MCVCMatrix[1][0], MCVCMatrix[2][0]);
+//  camera up
+vec3 vv = vec3(MCVCMatrix[0][1], MCVCMatrix[1][1], MCVCMatrix[2][1]);
+// camera direction
+vec3 ww = vec3(MCVCMatrix[0][2], MCVCMatrix[1][2], MCVCMatrix[2][2]);
+
+// create view ray
+vec3 rd = normalize( point.x*-uu + point.y*-vv + 7*ww);
```

Comment:
this block can be removed. You do not use it, let's keep it simple

Reply:
Fixed